Gems/Prefab/PrefabBuilder/PrefabGroup/PrefabGroupBehavior.cpp

```diff
@@ -215,6 +214,28 @@ namespace AZ::SceneAPI::Behaviors
         return Events::ProcessingResult::Success;
     }
 
+    Events::ProcessingResult PrefabGroupBehavior::ExportEventHandler::UpdateManifest(
```

Comment:
so the logic is now "create the prefab group" but not the mesh groups for the prefab until InitializeObject() ?

Reply:
I didn't change the existing logic. The mesh groups were previously only created when the manifest was created.

Now, in the UI, they get created also when you add a default prefab if there was none previously. It's purely visual for the UI, they were getting created before when the scene settings file was processed after saving, you just didn't see them in the UI until you closed/reopened it.